mbtools/copy_resources_s3.py

```diff
@@ -81,12 +104,14 @@ def resource_hashes(resource_dir):
                 duplicates.append({
                     'path': full_path,
                     'mime_type': mime_type,
-                    'sha1': sha1_key
+                    'sha1': sha1_key,
+                    'timestamp': datetime.now(timezone.utc).isoformat()
```

Comment:
Minor: Is there ever an expected code path where this doesn't end up being modified later? If not, maybe we should set to `None` or something here if we're just trying to initialize the field :man_shrugging: ?

Reply:
I thought about setting that to `none` but in the test `test_upload_resources` we use the `resource_hashes` method alone to build the `sha1_map` to compare with the data that ends up in the HTML file. Since the timestamp ends up being modified later in the `upload_resources` method. it causes an issue.